composer.json

```diff
@@ -14,8 +14,9 @@
         "phpmyadmin/sql-parser": "^5.2.0"
     },
     "require-dev":  {
-        "phpunit/phpunit": "^8.1",
-        "php-mock/php-mock": "^2.1"
+        "php-mock/php-mock": "^2.2",
+        "phpunit/phpunit": "^8.5",
+        "squizlabs/php_codesniffer": "^3.5"
```

Comment:
i don't think we need this , but it is also ok to leave it because it is required dev

Reply:
IMO it is important to be here, so the library is self-contained with his dependencies and we ensure all devs are using the same version of codesniffer to format, etc.